### PR TITLE
Add a note for GMT and SAC installation path

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/gmt/index.rst [hunk 1]
```diff
@@ -16,3 +16,8 @@ GMT
 
 GMT，全称 Generic Mapping Tools，中文一般译为“通用制图工具”，
 是地球科学最广泛使用的制图软件之一，其具有强大的绘图功能和数据处理功能。
+
+.. note::
+
+   我们建议将 GMT 安装到 :file:`~/opt/` 目录下，详情见
+   `《软件安装实践经验》 <https://seismo-learn.org/seismology101/best-practices/software-installation/>`__。
```
Comment (core-man) [inline source/gmt/index.rst:23]: We may need to use crossref insteas of url.

File: source/sac/index.rst [hunk 2]
```diff
@@ -18,3 +18,8 @@ SAC，全称 Seismic Analysis Code，是天然地震学领域使用最广泛的
 我们推荐使用中文手册学习 SAC。至少在某些章节，中文手册比官方手册更好。
 中文手册提供了更多有关数据处理的详细说明，以帮助初学者正确使用 SAC 和
 学习地震学数据处理基础。
+
+.. note::
+
+   我们建议将 SAC 安装到 :file:`~/opt/` 目录下，详情见
+   `《软件安装实践经验》 <https://seismo-learn.org/seismology101/best-practices/software-installation/>`__。
```
Comment (core-man) [inline source/sac/index.rst:25]: We may need to use crossref insteas of url.

File: source/gmt/index.rst [hunk 3]
```diff
@@ -16,3 +16,8 @@ GMT
 
 GMT，全称 Generic Mapping Tools，中文一般译为“通用制图工具”，
 是地球科学最广泛使用的制图软件之一，其具有强大的绘图功能和数据处理功能。
+
+.. note::
+
+   我们建议将 GMT 安装到 :file:`~/opt/` 目录下，详情见
```
Comment (core-man) [inline source/gmt/index.rst:22]: This is for installation from source code and packet file, right?

Comment (seisman) [reply source/gmt/index.rst:22]: Yes. Otherwise, gmt is installed into the default path.

Comment (core-man) [inline source/gmt/index.rst:22]:
```suggestion
   若选择编译源码安装 GMT，我们建议将 GMT 安装到 :file:`~/opt/` 目录下，详情见
```

Comment (core-man) [inline source/gmt/index.rst:22]:
```suggestion
   若选择编译源码安装 GMT，我们建议将 GMT 安装到 :file:`~/opt/` 目录下，详情见
```

**Review by core-man — APPROVED** [review]

File: source/gmt/index.rst (hunk 1 above)
Comment (seisman) [reply source/gmt/index.rst:23]: Will work on it in a separate PR.

**Pull request merged**